types/src/on_chain_config/vm_publishing_option.rs

```diff
@@ -4,14 +4,15 @@
 use crate::{on_chain_config::OnChainConfig, transaction::SCRIPT_HASH_LENGTH};
 use anyhow::{format_err, Result};
 use serde::{Deserialize, Serialize};
+use std::default::Default;
 
 /// Defines and holds the publishing policies for the VM. There are three possible configurations:
 /// 1. No module publishing, only allowlisted scripts are allowed.
 /// 2. No module publishing, custom scripts are allowed.
 /// 3. Both module publishing and custom scripts are allowed.
 /// We represent these as an enum instead of a struct since allowlisting and module/script
 /// publishing are mutually exclusive options.
-#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
```

Comment:
Can we avoid this default by changing the previous api to be `Option<VMPublishingOption>`?